_gsocprojects/2020/project_AstroLab.md

```diff
@@ -0,0 +1,11 @@
+---
+project: AstroLab
+layout: default
+logo: Astrolab-logo.png
+description: |
+  [AstroLab Software](https://astrolabsoftware.github.io/) is a project from [IJCLab](https://www.lal.in2p3.fr/en/) aiming at providing advanced software tools to overcome modern science challenges faced by research groups, and allow research communities to more fully exploit the big data ecosystem tools.
+summary: |
```

Comment:
when (summary == description) no need to add it.

Reply:
Thanks for the tip @agheata !